Cdnfsd_TransparentCDN_Popup_View_Stacks.php

```diff
@@ -0,0 +1,50 @@
+<?php
+namespace W3TC;
+
+if ( !defined( 'W3TC' ) )
+	die();
+?>
+<form class="w3tc_cdn_stackpath2_fsd_form" method="post">
```

Comment:
This file has a couple of references to "stackpath2". Please update all files in this pull request to reference "transparent", to avoid conflicts.